ftw/news/browser/resources/archive.scss

```diff
@@ -15,21 +19,9 @@
         display: block;
       }
     }
-
-    .count {
-      @include label-round();
-    }
   }
 
   .months {
     display: none;
-    .count {
-      &:before {
```

Comment:
@bierik, you are undoing your own code, which has been added for accessibility reasons in https://github.com/4teamwork/ftw.news/commit/76bc80e2bd88bda478eb94ec976629d207bc37f6#diff-c7810d83191d6a9bd57f25552be7a8f3R27

Is it a good idea to remove it again?